app/src/main/res/layout/fragment_add_doctor.xml

```diff
@@ -0,0 +1,174 @@
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/field_doctor_email">
+
+                    <com.google.android.material.textfield.TextInputEditText
+                        android:id="@+id/et_doctor_contact"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:afterTextChanged="@{(text) -> viewModel.isValidContact(text)}"
+                        android:hint="@string/hint_doctor_contact"
+                        android:imeOptions="actionDone"
+                        android:inputType="number"
+                        android:maxLength="10"
+                        android:maxLines="1"
+                        android:text="@={viewModel.doctorContactNumber}"
+                        android:textColor="@color/black"
+                        android:textSize="14sp" />
+
+                </com.google.android.material.textfield.TextInputLayout>
+
+                <androidx.appcompat.widget.AppCompatTextView
+                    android:id="@+id/txt_doctor_notification"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/margin_20"
+                    android:layout_marginTop="@dimen/margin_10"
+                    android:text="@string/doctor_notification_enable"
+                    android:textColor="@color/black"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/field_doctor_contact" />
+
+                <androidx.appcompat.widget.AppCompatToggleButton
+                    android:id="@+id/toggle_doctor_notification"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/margin_20"
+                    android:layout_marginTop="@dimen/margin_10"
+                    android:checked="@={viewModel.toggleLiveData}"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/field_doctor_contact" />
+
+                <androidx.appcompat.widget.AppCompatButton
+                    android:id="@+id/button_submit"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="@dimen/margin_10"
+                    android:enabled="@{viewModel.isDataValid}"
+                    android:onClick="@{()->viewModel.addDoctorData()}"
+                    android:text="@string/submit"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
```

Comment:
if you are using layout_constraintStart_toStartOf then please define layout_width 0dp

```
    android:layout_width="0dp"
        app:layout_constraintEnd_toEndOf="parent"
                    app:layout_constraintStart_toStartOf="parent"
```